vars/prod-vars.yaml

```diff
@@ -125,7 +125,81 @@ clusters:
           6564
         tlsClientConfig:
           insecure: true
-
+  - name: cluster-apps.ocp.prod.psi.redhat.com
+    data:
+      name: apps.ocp.prod.psi.redhat.com
+      namespaces:
+        - dtuchyna-thoth-dev
+        - fmurdaca-thoth-dev
+        - fpokorny-thoth-dev
+        - goern-thoth-dev
+        - jupyterhub-shuels-thoth-integration
+        - kpostlet-thoth-dev
```

Comment:
@goern are these namespaces also gonna manage by argo?

Reply:
Hmm, having the option to do so is not bad ;)